cypress/e2e/ddt_account_balance_tests.cy.js

```diff
@@ -0,0 +1,61 @@
+import { faker } from "@faker-js/faker";
+import { LoginApi } from "../api/login_api";
+import { LoginPage } from "../page-objects/login_page";
+import { HomePage } from "../page-objects/home_page";
+import accountBalanceData from "../fixtures/accounts_data.json";
+import { AccountApi } from "../api/account_api";
+/*Bug in app: POST API https://tegb-backend-877a0b063d29.herokuapp.com/tegb/accounts/create
```

Comment:
Je možné udělat v JSON disabled element s popisem, proč není kombinace testovaná a pak zde udělat if(), který by toto řešil. Toto ale nebylo vůbec vyžadované, jen posílám inspiraci :-)

Mohlo by to vypadat třeba takto:
**JSON:**
```json
  {
    "row": 4,
    "accountName": "Test_Account_",
    "balance": "1234567890.00",
    "disabled": true,
    "disabledReason": "BUG123 - 500 internal server error, in account creation"
  }
```

**Test:**
``` javascript
import accountsData from "../fixtures/account_balance_data.json";

describe("Disabled exampled", () => {
  accountsData.forEach((accountData) => {
    if (!accountData.disabled) {
      it(`create account no. ${accountData.row + 1}`, () => {
        // test code
      });
    } else {
      it.skip(`${accountData.disabledReason} - DDT Test`, () => {
        // Tady nic nedelame, jenom vypisujeme, ze test je disabled
      });
    }
  });
});
```


A výsledek v Cypress
![image](https://github.com/deni-adam/cypress-bank-app/assets/95618620/d2395c6c-7cb4-4d9b-80e4-1c42aac996c8)